src/clients/repositories.rb

```diff
@@ -641,10 +642,20 @@ def buildList
             @displayed_service == Ops.get_string(srv_state, "alias", "")
         )
         ret = Builtins.add(ret, t)
-      end 
+      end
 
+      # there is some service, so allow to filter repos without service (bnc#944504)
+      if ret.size > 2
+        t = Item(
+          Id(:without_service),
+          _("Without Service"),
```

Comment:
And then definitely add # TRANSLATORS: Explanation what it actually means :) and where it is used